src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/PileupSummary.java

```diff
@@ -107,6 +141,31 @@ public static void writeToFile(final String sample, final List<PileupSummary> re
         }
     }
 
+    public static class PileupSummaryComparator implements Comparator<PileupSummary> {
+        final SAMSequenceDictionary sequenceDictionary;
+        final List<String> contigsInOrder;
+
+        public PileupSummaryComparator(final SAMSequenceDictionary sequenceDictionary){
+            this.sequenceDictionary = sequenceDictionary;
+            contigsInOrder = sequenceDictionary.getSequences().stream()
+                    .map(ssr -> ssr.getSequenceName())
+                    .collect(Collectors.toList());
+        }
+
+        @Override
+        public int compare(PileupSummary ps1, PileupSummary ps2) {
+            // Use Contig Index in case the contig name is e.g. chr2
+            final int contigIndex1 = contigsInOrder.indexOf(ps1.getContig());
+            final int contigIndex2 = contigsInOrder.indexOf(ps2.getContig());
+
+            if (contigIndex1 != contigIndex2){
+                return contigIndex1 - contigIndex2;
```

Comment:
`Integer.compare` is clearer than subtraction.

Reply:
Nice catch. Done